### PR TITLE
Config per environment

### PR DESCRIPTION
**Pull request by igalarzab**

### CODE REVIEW

File: manifests/configmap.ee.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+kind: ConfigMap
+metadata:
+  name: backend-config
+  namespace: :env:
+  labels:
+    app: backend
+    environment: :env:
+data:
+  AWS_REGION: eu-west-1
+  AUTH_MAGIC_LINK_ENABLED: "false"
```
Comment (Tankenstein) [inline manifests/configmap.ee.yaml:13]: should these not just be false and true, without the quotes?

File: manifests/configmap.ee.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  AUTH_ESTONIAN_ID_ENABLED: "true"
```
Comment (oliverviljamaa) [inline manifests/configmap.ee.yaml:14]: Could we also add `DEFAULT_LANGUAGE_CODE` to both?

Comment (oliverviljamaa) [reply manifests/configmap.ee.yaml:14]: (`"et"` and `"en"`)

Comment (Tankenstein) [reply manifests/configmap.ee.yaml:14]: We can add it ourselves in another pr

**Pull request merged**